packages/client/components/ScopingSearchInput.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {commitLocalUpdate} from 'react-relay'
+import useAtmosphere from '../hooks/useAtmosphere'
+import {PALETTE} from '../styles/paletteV3'
+import Icon from './Icon'
+
+const SearchInput = styled('input')({
+  appearance: 'none',
+  border: 'none',
+  borderLeft: `1px solid ${PALETTE.SLATE_400}`,
+  color: PALETTE.SLATE_700,
+  fontSize: 16,
+  margin: 0,
+  padding: 12,
+  outline: 0,
+  backgroundColor: 'transparent',
+  width: '100%'
+})
+
+const Wrapper = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  flex: 1
+})
+
+const ClearSearchIcon = styled(Icon)<{isEmpty: boolean}>(({isEmpty}) => ({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  padding: 12,
+  visibility: isEmpty ? 'hidden' : undefined
+}))
+
+interface Props {
+  placeholder: string
+  queryString: string
+  meetingId: string
+  linkedRecordName: string
+}
+
+const ScopingSearchInput = (props: Props) => {
```

Comment:
+1 same here. I think we should either use the `Jira` prefix, or implement this in GitHub & GitLab, or create an issue to do so

Reply:
Created an issue https://github.com/ParabolInc/parabol/issues/6435